backend/src/main/java/org/globe42/CrypticApplication.java

```diff
@@ -2,11 +2,12 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ConfigurableApplicationContext;
 
 @SpringBootApplication
 public class CrypticApplication {
 
 	public static void main(String[] args) {
-		SpringApplication.run(CrypticApplication.class, args);
+		ConfigurableApplicationContext run = SpringApplication.run(CrypticApplication.class, args);
```

Comment:
No. I'll remove that. I did that to test my first DAO before I actually added unit tests.